### ISSUE TITLE
Lizzie does not load on Win 10

### ISSUE THREAD

**Issue by F0re** [featurecat/lizzie#344]
Hi, I'm new to lizzie and just installed 0.5 version.
What happens is that the UI windows briefly flashes and then closes. This happens regardless if I run CPU or GPU version.

If I run it from command line with "java -jar Lizzie.jar" then it just says "Leelaz process ended."

I installed Java 8 and Lizzie, but haven't installed any other additional libraries or software.

Running Windows 10 Home version 1803 (17134.165) 
CPU: Intel(R) Core(TM) i5 CPU 760 @ 2.80 GHz

**Comment by wonsiks** [comment]
try this
https://www.microsoft.com/en-us/download/details.aspx?id=48145

**Comment by featurecat** [comment]
You're right. I'm so lazy hahaha. My work ends on August 17. I'd like to take some time to improve and update Lizzie based on the last few months of issues.
